### PR TITLE
LPS-138139 Add missing key

### PR DESCRIPTION
**Pull request by julien**
As described in [LPS-138139](https://issues.liferay.com/browse/LPS-138139), 
a language key was missing for the button's tooltip

**Test plan**
Fetch this branch
Deploy the `remote-app-admin-web` OSGi module
Assert that the tooltip renders the correct key

**Before**

**After**

### CODE REVIEW

File: modules/apps/remote-app/remote-app-admin-web/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -1,3 +1,4 @@
+add-remote-web-app=Add Remote Web App
```
Comment (diegonvs) [inline modules/apps/remote-app/remote-app-admin-web/src/main/resources/content/Language.properties:1]: It looks fine to me.

However, why not `new-remote-app` as it was already defined?

Comment (markocikos) [reply modules/apps/remote-app/remote-app-admin-web/src/main/resources/content/Language.properties:1]: I'd say that's up to the product team. If they think a slightly different label better explains the feature, :+1:

**Review by markocikos — APPROVED** [review]
LGTM!

**Comment by julien** [comment]
@markocikos of course, in that case (but let me check) I think we're the ones responsible for this module so the debate is open. My intent was to solve this issue since it seemed like an easy fix.

**Comment by julien** [comment]
@diegonvs @markocikos sorry guys, got confused once again by reading comments on this thread from bottom to top :facepalm: 

@diego to answer your question, that key is used for the title 



I thought having a different value for the tooltip made more sense, however this can be changed if we think it makes more sense.

**Comment by diegonvs** [comment]
> to answer your question, that key is used for the title

Nice :)

**Comment by markocikos** [comment]
>I thought having a different value for the tooltip made more sense, however this can be changed if we think it makes more sense

@julien  Looking at tooltips of [+] icons in DXP, they are about 50% labeled `New`, 50% `Add X` :joy:  I guess you can flip a coin on this one.

**Comment by julien** [comment]
@markocikos I honestly don't have any preference about this, but you're right. I think this is acceptable :joy:

**Comment by bryceosterhaus** [comment]
ci:forward

**Pull request closed without merging**